pkg/synthetictests/platformidentification/types.go

```diff
@@ -35,6 +35,7 @@ type ClusterData struct {
 	CloudRegion           string
 	CloudZone             string
 	ClusterVersionHistory []string
+	MasterNodesUpdated    string
```

Comment:
Looks like we only ever store Y/N here? This should probably be a boolean field, unless you plan to have a third option or add more context in the string.

Reply:
Originally it was a boolean but there is a case where we don't have the intervals to compare against so at first I went with 'U' for unknown.  But then since in bigquery existing runs will have it missing I changed the default to "".

https://github.com/openshift/origin/blob/c5f09b081e421519283e593c839f0d14ae77a084/pkg/test/ginkgo/cmd_runsuite.go#L482

and

https://github.com/openshift/origin/blob/c5f09b081e421519283e593c839f0d14ae77a084/test/extended/util/disruption/disruption.go#L174

Reply:
Ah ok, that makes sense then, there's 3 values, thanks